packages/core/android/src/main/java/com/segment/analytics/reactnative/core/RNAnalyticsModule.kt

```diff
@@ -245,7 +245,11 @@ private fun optionsFrom(context: ReadableMap?, integrations: ReadableMap?): Opti
     }
 
     integrations?.toHashMap()?.forEach { (key, value) ->
-        options.setIntegration(key, value.toString().toBoolean())
+        if (value is HashMap<*, *>) {
+            options.setIntegrationOptions(key, value.toMap() as Map<String, Object>)
```

Comment:
@squirly could you change this to use `Any` rather than `Object` since its more appropriate for Kotlin

Reply:
```suggestion
            options.setIntegrationOptions(key, value.toMap() as Map<String, Any>)
```